src/junit/LIC_test.java

```diff
@@ -23,14 +22,14 @@ public void cmv_getter_test(){
     public void checkIfValidGivesTrue(){
         int[][] datapoints = {{4,0},{3,0},{2,0},{1,0},{0,0}};
         CMV cmv = new CMV(datapoints);
-        assertTrue(cmv.get_cmv_value(5));
+        assertTrue(cmv.lic5_calculate());
```

Comment:
This causes an error due to function is private. There must been a mistake by removing the functioning code `cmv.get_cmv_value(5)`.